### PR TITLE
Update sso-saml.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Reviewed by Lindsay, Corey and Alexis via proposed changes on [Google Doc](https://docs.google.com/document/d/1F9F085I1Ih2zFo0lyXJ5ndDMwIBBp42-U53vwR2Peyg/edit?usp=sharing )

### CODE REVIEW

File: source/deployment/sso-saml.rst [hunk 1]
```diff
@@ -8,11 +8,15 @@ Mattermost can be configured to act as a SAML 2.0 Service Provider. The SAML Sin
...
+Mattermost officially supports Okta, OneLogin and Microsoft ADFS as the identity providers (IDPs), please see links below for more details on how to configure SAML with these providers. 
 
 .. toctree::
   :titlesonly:
 
   Okta SAML Configuration <sso-saml-okta>
   OneLogin SAML Configuration <sso-saml-onelogin.rst>
   Microsoft ADFS SAML Configuration <sso-saml-adfs>
+
+If you'd like, you may also try configuring SAML for a custom IDP.  For instance, customers have successfully set up Duo and PingFederate as a custom IDP. We are happy to provide assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues. Please see more information on getting support `here <https://mattermost.com/support/>`_.
```
Comment (jasonblais) [inline source/deployment/sso-saml.rst]: > Duo and PingFederate as a custom IDP

SimpleSAMLphp was also configured successfully

> We are happy to provide assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues

The language makes it sound like we'll work with them until the issues are resolved. Not sure if there is a standard / expectation of whether we'd resolve all issues, and the cadence of it.

Comment (wiersgallak) [reply source/deployment/sso-saml.rst]: Agreed, we can be a bit more explicit here as some issues will be out of our ability to resolve directly.  We could add "as they relate to Mattermost SAML configuration settings." Open to other suggestions as well.

Comment (jasonblais) [reply source/deployment/sso-saml.rst]: How about.

```
We are open to providing assistance when configuring your custom IDP by answering Mattermost technical configuration questions and working with your IDP provider in support of resolving issues as they relate to Mattermost SAML configuration settings. However, we cannot guarantee your connection will work with Mattermost.

Please see more information on getting support `here <https://mattermost.com/support/>`_ and submit requests for official support of a particular provider on our `feature idea forum <https://mattermost.uservoice.com>`_.
```

Comment (wiersgallak) [reply source/deployment/sso-saml.rst]: Thanks, Jason -this proposal makes sense to me

**Pull request merged**